tasks/Module 3/task_2.py

```diff
@@ -0,0 +1,21 @@
+"""My split function."""
+
+
+def my_split(source_string, separator):
```

Comment:
The task here was to mimic the behavior of the `str.split`. 

`str.split` has `separator=None`, and if the separator was not provided, then it uses empty space `' '` as a default separator.

Also, `split` raises `ValueError` if someone provided an empty string as a separator